src/python/grpcio/grpc/__init__.py

```diff
@@ -298,6 +298,33 @@ class Status(six.with_metaclass(abc.ABCMeta)):
 class RpcError(Exception):
     """Raised by the gRPC library to indicate non-OK-status RPC termination."""
 
+    def trailing_metadata(self):
+        """Accesses the trailing metadata sent by the server.
+
+        This method blocks until the value is available.
+
+        Returns:
+          The trailing :term:`metadata`.
+        """
```

Comment:
We are adding these methods, and they may or may not be implemented. In current code base, there are corner cases of a `grpc.RpcError` is not a `grpc.Call`. So, I would suggest to raise a `NotImplementedError` by default.